consensus/src/chained_bft/common.rs

```diff
@@ -49,3 +55,12 @@ impl<T> Payload for T where
         + 'static
 {
 }
+
+pub fn round_hash(round: Round) -> HashValue {
```

Comment:
this probably should be `impl Cryptohash for Round`

Reply:
Well, we can't `impl CryptoHash` for a type alias. Unfortunately, today `Round` is simply a type alias to `u64`, which means we can't add implementations to it.
We will have to change `Round` to work properly with reconfiguration anyway, but I think this change deserves a separate PR of its own.

Reply:
This is unfortunate.  Here's an example of how it could work if not for SimplerSerializer (https://gist.github.com/aching/823d065b5e25096b64a709e7f7b7c771).  

This is fine for this PR but perhaps we can consider moving the CryptoHash definition into the types crate to make this possible with SimplerSerializer.